server/sql/dev/create-tables-dev.sql

```diff
@@ -1,18 +1,47 @@
 \c socobo_dev;
 
+--
+--  SOCOBO USER
+--
+
+CREATE TABLE Socobo_User_Role (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR NOT NULL UNIQUE,
+  created NUMERIC NOT NULL,
+  lastModified NUMERIC NOT NULL CHECK (lastModified >= created)
+);
+
+CREATE TABLE Socobo_User_Provider (
+  id SERIAL PRIMARY KEY,
```

Comment:
I think my main problem is that I don't really get the point of storing this kind of things (roles, providers) to the db? What is the benefit? Whats the problem with simply using a enum for roles and providers and checking against these values?

Do we really use the values from the db or do we only check if the id in the user entity has a certain value?